src/libmagickwand.jl

```diff
@@ -138,37 +138,65 @@ const NoCompression = 1
 
 type MagickWand
     ptr::Ptr{Void}
+
+    function MagickWand()
+        ptr = ccall((:NewMagickWand, libwand), Ptr{Void}, ())
+        obj = new(ptr)
```

Comment:
should probably add `ptr == C_NULL && throw(OutOfMemoryError())`    and same below?